packages/plugin-e2e/src/models/pages/PanelEditPage.ts

```diff
@@ -185,4 +182,34 @@ export class PanelEditPage extends GrafanaPage {
 
     return responsePromise;
   }
+
+  /** Return page object for the panel edit options group with the given label */
+  getCustomOptions(label: string): PanelEditOptionsGroup {
+    const locator = this.getOptionsGroupLocator(label);
+    return new PanelEditOptionsGroup(this.ctx, locator, label);
+  }
+
+  getStandardOptions(): PanelEditOptionsGroup {
+    return this.getCustomOptions('Standard options');
+  }
+
+  getValueMappingOptions(): PanelEditOptionsGroup {
+    return this.getCustomOptions('Value mappings');
+  }
+
+  getDataLinksOptions(): PanelEditOptionsGroup {
+    return this.getCustomOptions('Data links');
+  }
+
+  getThresholdsOptions(): PanelEditOptionsGroup {
+    return this.getCustomOptions('Thresholds');
+  }
+
+  private getOptionsGroupLocator(label: string): Locator {
+    if (semver.gte(this.ctx.grafanaVersion, '11.4.0')) {
+      return this.ctx.page.getByTestId(`data-testid Options group ${label}`);
```

Comment:
Change to `this.getByGrafanaSelector(selectors.components.OptionsGroup.group(label))`